README.md

```diff
@@ -34,6 +34,7 @@ _Bad software is everywhere, and we're tired of it. Sentry is on a mission to he
 - RAM bundle support
 - Hermes support
 - Expo support ([sentry-expo](https://github.com/expo/sentry-expo))
+- RN New Architecture support
```

Comment:
Is it obvious for react native peepz what "New Architecture" means?

Reply:
Yes, definitely, it's not only new with lowercase n but also that's how the RN team calls it "New Architecture".